tools/purge_ems.rb

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env ruby
+enviroment = File.expand_path('../config/environment', __dir__) + ".rb"
+require enviroment if File.exist?(enviroment)
+
+if !defined?(Vmdb::Application)
+  warn "This script must exist in the tools directory or can live elsewhere but needs to be run via rails runner path/to/purge_ems.rb --id 1234 --follow true"
```

Comment:
Since we are shipping this, less choices is better.  No need to state in can run elsewhere.  Something like:

```suggestion
  warn "This script must be run via rails runner: `rails runner path/to/purge_ems.rb --id 1234 --follow true`"
```

Reply:
In fact, just drop it entirely, and assume it's run from there - it can fail otherwise.  All of the other tools scripts do this.

Reply:
you can't if you don't have it there yet... for users on podified who don't have it in tools, they likely can't put it there

Reply:
The intent is to support existing versions on podified as they will not have this in the tools directory and can't move it there.

Reply:
Could we just tell them to put it in the tools directory on podified?  We don't have a RO filesystem yet and by the time we do we will likely have this committed anyway

Reply:
I don't know.  I thought we couldn't write to anything below `/var/www/miq/vmdb`.  I'll have to test on podified.

Reply:
It would be ideal to not need to support rails runner

Reply:
```
sh-4.4$ cat BUILD 
20230901001727_b7b3552
sh-4.4$ echo "test" > tools/test.rb
sh-4.4$ cat tools/test.rb 
test
```

Reply:
> I don't know. I thought we couldn't write to anything below /var/www/miq/vmdb. I'll have to test on podified.

You can definitely write to the directory.  It just won't persist between pod restarts, but if there's a pod restart it will have to be re-run anyway.

Reply:
Thanks, I tested it too. Looks like we can write but yes, it will disappear when the pod restarts.

```
sh-4.4$ cp create_local_yum_repo.sh /var/www/miq/vmdb/tools
sh-4.4$ ls -altr /var/www/miq/vmdb/tools/create_*
-rwxrwxr-x. 1 root       root 1475 Aug 31 10:29 /var/www/miq/vmdb/tools/create_evm_snapshot.rb
-rwxr-xr-x. 1 1000680000 root  337 Sep  1 15:52 /var/www/miq/vmdb/tools/create_local_yum_repo.sh
```

It's much simpler to assume we're living in tools for either runner or ruby invocation.